### PR TITLE
update the homepage takeover to iot business whitepaper

### LINKED ISSUE #2795: Update the homepage takeover to iot business whitepaper
I was wondering if we could switch on the IoT business model whitepaper takeover until the 19th?

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- update the homepage takeover to iot business whitepaper

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
  - See that the takeover is now the IoT ‘How to pick a winning IoT business model’ one
  - See that you can download the whitepaper

## Issue / Card

Fixes #2795

## Screenshots
image

### CODE REVIEW

**Comment by frankiegranato** [comment]
image
I realise that this is probably not within our remit (because its built in marketo), but for what its worth, the landing page doesn't have any padding under the button. See screen shot.

**Comment by frankiegranato** [comment]
The takeover looks good to me. 
To avoid repetition, lets also change the take-under. I'll have a look into what we swap it for (and maybe check in with Thibaut) and file and issue :-)

**Comment by pmahnke** [comment]
@frankiegranato I have added one that is the product month

File: templates/takeovers/takeunders/_ubuntu-product-month.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div class="col-6 p-takeunder" style="background-color: #111111;">
+  <div class="row">
+    <div class="u-equal-height">
+      <div class="col-4 u-align--center u-vertically-center u-hide--small">
+        <img src="https://assets.ubuntu.com/v1/b0bd120e-UPM-Illustration.svg" width="180" />
+      </div>
+      <div class="col-8">
+        <h3><a href="https://pages.ubuntu.com/ubuntu_product_month.html" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Ubuntu Product Month takeunder', 'eventLabel' : 'Find out more', 'eventValue' : undefined });"></a>Ubuntu product month omnibus&nbsp;&rsaquo;</a></h3>
```
Comment (Caleb-Ellis) [inline templates/takeovers/takeunders/_ubuntu-product-month.html]: There's an extra `</a>` in here that closes the link early

File: templates/takeovers/takeunders/_ubuntu-product-month.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div class="col-6 p-takeunder" style="background-color: #111111;">
+  <div class="row">
+    <div class="u-equal-height">
+      <div class="col-4 u-align--center u-vertically-center u-hide--small">
+        <img src="https://assets.ubuntu.com/v1/b0bd120e-UPM-Illustration.svg" width="180" />
```
[the remark this answers is not in the document]
Comment (Caleb-Ellis) [reply templates/takeovers/takeunders/_ubuntu-product-month.html]: @pmahnke You've made accesslint bot sad :(

**Pull request merged**